src/JBrowse/QueryParamConfigMapper.js

```diff
@@ -59,7 +59,7 @@ define(['dojo/_base/declare', 'JBrowse/Util/dot-object'], function (declare, dot
             var storeTracks = {};
             var storeBookmarks = {};
             Object.keys(queryParams).forEach(function (queryParam) {
-                if (queryParam.indexOf('addStore\.') == 0) {
+                if (queryParam.indexOf('addStores\.') == 0) {
```

Comment:
Could you add a block comment to the top of this file saying what this class does?  For posterity.

Reply:
added doc at the top.  Thanks.